sample-app/notification-sample-code/nodejs/README.md

```diff
@@ -0,0 +1,121 @@
+# Notification Sample Code
+
+This sample code makes use of the Create Subscription endpoint to create a subscription for ringing events on a range of endpoints defined by the user. This sample code is expected to output any received webhooks to the console
+
+## Pre-requisites
+
+[Node.js](https://nodejs.org/en/) - Install node.js
+
+
+## Setup
+
+At the root level, run the below command to install the necessary dependencies
+
+```
+$ npm install
+```
+
+# Provide valid input data
+
+Open input inputData.js file and modify the below details
+
+```json
+{
+    userdata: {
+        response_type: "code",       // no change needed
+        username: "USER_NAME",       // provide your username for user 1
+        password: "PASSWORD",        // provide password for user
+        account_id: "ACCOUNT_ID",    // provide account_id for user
+    },
+    appData: {
+        client_id: "CLIENT_ID"       // provide valid unique clientId issued by Mitel
+    },
+    // Data for subscription creation
+    subscriptionData: {
+        applicationId: "APPLICATION_ID", // 128-bit GUID that identifies the application for trust/signing purposes
```

Comment:
I got confused over what an APPLICATION_ID was, we should be more specific that this is an ID that you specify.